apps/vs-code-designer/src/app/commands/createNewProject/createProjectSteps/ScriptProjectCreateStep.ts

```diff
@@ -43,12 +43,14 @@ export class ScriptProjectCreateStep extends ProjectCreateStepBase {
     }
 
     const localSettingsJsonPath: string = path.join(context.projectPath, localSettingsFileName);
+    //added new setting here to allow local start without SAS
     if (await confirmOverwriteFile(context, localSettingsJsonPath)) {
       const localSettingsJson: ILocalSettingsJson = {
         IsEncrypted: false,
         Values: {
           AzureWebJobsStorage: '',
           APP_KIND: logicAppKind,
+          'AzureFunctionsJobHost__extensions__workflow__Settings__Runtime.Triggers.AnonymousAuthEnabledForDevEnvironment': 'true',
```

Comment:
nit: the formatting here is off

Reply:
Also make sure to just add it when is needed for static web apps and not for all of the scenarios